### ISSUE TITLE
plot_random_forest: test data leakage in DSR train set

### ISSUE THREAD

**Issue by ngoix** [Menelau/DESlib#126]
I think
```
X_train, X_dsel, y_train, y_dsel = train_test_split(X, y, test_size=0.50,
                                                    random_state=rng)
```
should be

```
X_train, X_dsel, y_train, y_dsel = train_test_split(X_train, y_train, test_size=0.50,
                                                    random_state=rng)
```

**Comment by luizgh** [comment]
Thanks for spotting this issue. Fixing it in 39b4da31f1ab5b788a2ae6eb2a4795e686148b5b

**Comment by luizgh** [comment]
Fixed in #127
